### PR TITLE
Improve documentation a bit in several places

### PR DESCRIPTION
**Pull request by cmarqu**
Refs #2354.

### CODE REVIEW

File: documentation/source/building.rst [hunk 1]
```diff
@@ -71,9 +74,10 @@ Cocotb
     Use this to override the default behavior of annotating cocotb output with
     ANSI color codes if the output is a terminal (``isatty()``).
 
-    ``COCOTB_ANSI_OUTPUT=1`` forces output to be ANSI regardless of the type of ``stdout`` or the presence of :envvar:`NO_COLOR`.
-
-    ``COCOTB_ANSI_OUTPUT=0`` suppresses the ANSI output in the log messages
+    ``COCOTB_ANSI_OUTPUT=1``
+       forces output to be ANSI regardless of the type of ``stdout`` or the presence of :envvar:`NO_COLOR`
+    ``COCOTB_ANSI_OUTPUT=0``
+       suppresses the ANSI output in the log messages
```
Comment (ktbarrett) [inline documentation/source/building.rst:80]: ANSI what? ANSI is an organization. Should be corrected to "ANSI colors" or "ANSI color codes"

File: documentation/source/building.rst [hunk 2]
```diff
@@ -177,6 +180,10 @@ Regression Manager
     .. deprecated:: 1.5
         :class:`cocotb.hook` is deprecated, and in the future this variable will be ignored.
 
+.. envvar:: COCOTB_PDB_ON_EXCEPTION
+
+   If defined, cocotb will drop into the Python debugger (:mod:`pdb`) if a test fails with an exception.
```
Comment (ktbarrett) [inline documentation/source/building.rst:185]: You mention later that this doesn't work on many simulators, maybe you should reiterate that here.

**Review by ktbarrett — APPROVED** [review]

File: documentation/source/building.rst (hunk 1 above)
Comment (cmarqu) [reply documentation/source/building.rst]: Old text, but yeah.

File: documentation/source/building.rst (hunk 2 above)
Comment (cmarqu) [reply documentation/source/building.rst:185]: Done.

File: documentation/source/building.rst (hunk 1 above)
Comment (ktbarrett) [reply documentation/source/building.rst]: I understand, good to fix it now before we forget.

**Pull request merged**